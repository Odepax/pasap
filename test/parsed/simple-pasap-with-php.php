<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Pasap Test</title>
	</head>
	<body>
		<h1>Amaze</h1>
		<list>
			<?php foreach ([ "One", "Two", "Three" ] as $item): ?>
				<list-item><?= $item ?></list-item>
			<?php endforeach; ?>
		</list>
	</body>
</html>
