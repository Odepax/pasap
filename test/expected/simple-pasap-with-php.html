<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Pasap Test</title>
	</head>
	<body>
		<h1>Amaze</h1>
		<ul>
			<li>One</li>
			<li>Two</li>
			<li>Three</li>
		</ul>
	</body>
</html>
